resqpy/grid.py

```diff
@@ -321,8 +319,7 @@ def denaturalized_cell_index(self, c0):
         return (k0, j0, i0)
 
     def denaturalized_cell_indices(self, c0s):
-        """Returns an integer numpy array of shape (..., 3) holding kji0 indices for the cells with given natural
-        indices.
+        """Returns an integer array holding kji0 indices for the cells with given natural indices.
 
         argument:
            c0s: numpy integer array of shape (...,) being natural cell indices (for a flattened array)
```

Comment:
Think the "3" is missing here
```suggestion
           c0s: numpy integer array of shape (..., 3) being natural cell indices (for a flattened array)
```